.github/workflows/ci_linux_x64_clang.yml

```diff
@@ -7,14 +7,36 @@
 name: CI - Linux x64 clang
 
 on:
+  push:
+    branches:
+      - main
```

Comment:
The build status badges I added at https://github.com/iree-org/iree?tab=readme-ov-file#build-status and https://iree.dev/developers/general/github-actions/#platform-builds are already reporting status for `push` events in this workflow ;)

We just haven't had any runs yet: https://github.com/iree-org/iree/actions/workflows/ci_linux_x64_clang.yml?query=branch%3Amain+event%3Apush . Once this is merged they should show the latest status.

Reply:
Oops, missed one. Updated it now :P